src/components/Main.js

```diff
@@ -120,6 +120,10 @@ class Main extends React.Component {
     }
   }
 
+  showDefaltMainMenuActions() {
```

Comment:
mne sa viac paci viac funkcii a mensi render.. ked je vsetko v render tak mi to silne pripomina programovanie v C ked vsetko bolo vo funkcii main :)